configs/default/AIKO-AIKONF7.config

```
@@ -1,8 +1,11 @@
 # Betaflight / STM32F7X2 (S7X2) 4.0.6 Sep  1 2019 / 01:10:08 (2a64051a2) MSP API: 1.41
+#mcu STM32F7X2
 
 #define USE_GYRO_SPI_MPU6000
 #define USE_ACC_SPI_MPU6000
 #define USE_ACCGYRO_BMI270
+#define USE_BARO_SPI_BMP280
+#define USE_BARO_DPS310
```

Board uses SPI for BARO
```suggestion
#define USE_BARO_SPI_DPS310
```